pinot-core/src/main/java/org/apache/pinot/core/segment/index/readers/text/LuceneTextIndexReader.java

```diff
@@ -121,7 +131,7 @@ public MutableRoaringBitmap getDocIds(Object value) {
     try {
       Query query = _queryParser.parse(searchQuery);
       _indexSearcher.search(query, docIDCollector);
-      return getPinotDocIds(docIDs);
+      return getPinotDocIdsFromMappingFile(docIDs);
     } catch (Exception e) {
       LOGGER.error("Failed while searching the text index for column {}, search query {}, exception {}", _column,
```

Comment:
You don't really need to log the error if you are going to throw out the exception. The catcher will log it with the stack trace

Reply:
Removed the e.getMessage() from log. The log is still needed though since I want to capture column name and search expression for the failed query